extensions-core/hdfs-storage/src/main/java/io/druid/storage/hdfs/HdfsStorageDruidModule.java

```diff
@@ -126,5 +130,18 @@ public void configure(Binder binder)
     Binders.taskLogsBinder(binder).addBinding("hdfs").to(HdfsTaskLogs.class);
     JsonConfigProvider.bind(binder, "druid.indexer.logs", HdfsTaskLogsConfig.class);
     binder.bind(HdfsTaskLogs.class).in(LazySingleton.class);
+
+    UserGroupInformation.setConfiguration(conf);
+    final String username = props.getProperty("druid.hdfs.security.kerberos.principal");
```

Comment:
@fjy not sure what need to be configurable ? like the property name `druid.hdfs.security.kerberos.principal` ?